ztp/source-crs/MachineConfigAcceleratedStartup.yaml

```diff
@@ -0,0 +1,88 @@
+        mode: 493
+        path: /usr/local/bin/accelerated-container-startup.sh
+    systemd:
+      units:
+      - contents: |
+          [Unit]
+          Description=Unlocks more CPUs for critical system processes during container startup
+
+          [Service]
+          Type=simple
+          ExecStart=/usr/local/bin/accelerated-container-startup.sh
+
+          [Install]
+          WantedBy=multi-user.target
+        enabled: true
+        name: accelerated-container-startup.service
+      - contents: |
+          [Unit]
+          Description=Unlocks more CPUs for critical system processes during container shutdown
+          DefaultDependencies=no
+
+          [Service]
+          Type=simple
+          ExecStart=/usr/local/bin/accelerated-container-startup.sh
+
+          [Install]
+          WantedBy=shutdown.target reboot.target halt.target
+        enabled: true
+        name: accelerated-container-shutdown.service
+---
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
There is no need to duplicate the MC under source-crs for Worker & Master the l`abel: machineconfiguration.openshift.io/role: worker|master  ` code be set in the PGT

Reply:
Is there an example of this somewhere?

Reply:
similar to the sctp MC here
https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/ztp-policy-generator/testPolicyGenTemplate/group-du-sno-ranGen.yaml#L45

Reply:
Awesome.  I think I've done a much better job this time :)